index.html

```diff
@@ -28,7 +28,7 @@ <h1>
     </nav>
     <main class="Main" main>
```

Comment:
class names in lower cases and the main> doesn't fit here because it isnt a self closing tag